src/core/scene/sortFunctions.js

```diff
@@ -0,0 +1,56 @@
+// Custom A-Frame sort functions.
```

Comment:
I think putting these into a new module is right.  Not sure whether this is the right place in the directory structure, or if there's a more suitable place.

I didn't think these were `utils` as they aren't part of the exposed A-Frame API.  COuldn't see another obvious directory to put them in, so put them here.

Reply:
I would put them in the renderer file where they are used. see `setupCanvas` function in a-scene https://github.com/aframevr/aframe/blob/master/src/core/scene/a-scene.js#L863 No need to prepend `aframe` to the function name

Reply:
We can think of modularizing later if the functions turn to be useful somewhere else (like those in utils)

Reply:
Are you sure about this?  They are used in both a-scene, and renderer.  Defining them in renderer means that a core module (a-scene) will have a dependency on a systems module (renderer).

That seems like the wrong direction for a dependency to me.

It might be preferable to define them in a-scene?

Agree with removing `aframe` from the function names, and have made that change.

Reply:
Sorry missed they're used in a-scene as well. Do we need them in both places? Can we consolidate in just one?

Reply:
I don't see the functions used in a-scene. I might be missing something?

Reply:
Sorry, my mistake. They were used in a-scene until I did some refactoring proposed by @mrxz .  I'll fix as you suggest now.